Resources/Prototypes/Entities/Objects/Fun/toys.yml

```diff
@@ -6,7 +6,12 @@
   name: bee plushie
   description: A cute toy that resembles an even cuter programmer.
   components:
-  - type: Toys
+  - type: EmitSoundOnUse
+    soundCollection: ToySqueak
+  - type: EmitSoundOnLand
+    soundCollection: ToySqueak
```

Comment:
this gets duplicated a lot so think about making a BaseToy prototype in YAML that defines the emitsounds/loopingsound and such and having all of these inherit from it

Reply:
good idea